dropwizard-app/src/test/java/org/coner/it/CreateRegistrationIntegrationTest.java

```diff
@@ -0,0 +1,194 @@
+        eventId = UnitTestUtils.getEntityIdFromResponse(addEventResponseContainer);
+    }
+
+    @Test
+    public void testGetZeroRegistrationsForValidEvent() {
+        Client client = IntegrationTestUtils.buildClient(RULE);
+        URI eventRegistrationsUri = IntegrationTestUtils.jerseyUriBuilderForApp(RULE)
+                .path("/events/{eventId}/registrations")
+                .build(eventId);
+
+        Response getRegistrationsResponseContainer = client.target(eventRegistrationsUri)
+                .request(MediaType.APPLICATION_JSON_TYPE)
+                .accept(MediaType.APPLICATION_JSON_TYPE)
+                .get();
+
+        assertThat(getRegistrationsResponseContainer.getStatus()).isEqualTo(HttpStatus.OK_200);
+
+        GetEventRegistrationsResponse getEventRegistrationsResponse = getRegistrationsResponseContainer
+                .readEntity(GetEventRegistrationsResponse.class);
+        assertThat(getEventRegistrationsResponse).isNotNull();
+        assertThat(getEventRegistrationsResponse.getRegistrations().size()).isZero();
+    }
+
+    @Test
+    public void whenCreateRegistrationItShouldPersist() {
+        Client client = IntegrationTestUtils.buildClient(RULE);
+        URI eventRegistrationsUri = IntegrationTestUtils.jerseyUriBuilderForApp(RULE)
+                .path("/events/{eventId}/registrations")
+                .build(eventId);
+
+        // Post first Registration
+        AddRegistrationRequest addRegistrationRequest0 = new AddRegistrationRequest();
+        addRegistrationRequest0.setFirstName(TestConstants.REGISTRATION_FIRSTNAME);
+        addRegistrationRequest0.setLastName(TestConstants.REGISTRATION_LASTNAME);
+
+        Response addRegistrationResponse0 = client.target(eventRegistrationsUri)
+                .request(MediaType.APPLICATION_JSON_TYPE)
+                .accept(MediaType.APPLICATION_JSON_TYPE)
+                .post(Entity.json(addRegistrationRequest0));
+
+        assertThat(addRegistrationResponse0.getStatus()).isEqualTo(HttpStatus.CREATED_201);
+        final String registrationId0 = UnitTestUtils.getEntityIdFromResponse(addRegistrationResponse0);
+
+        // Post second Registration
+        AddRegistrationRequest addRegistrationRequest1 = new AddRegistrationRequest();
+        addRegistrationRequest1.setFirstName(TestConstants.REGISTRATION_FIRSTNAME + "-2");
+        addRegistrationRequest1.setLastName(TestConstants.REGISTRATION_LASTNAME + "-2");
+
+        Response addRegistrationResponse1 = client.target(eventRegistrationsUri)
+                .request(MediaType.APPLICATION_JSON_TYPE)
+                .accept(MediaType.APPLICATION_JSON_TYPE)
+                .post(Entity.json(addRegistrationRequest1));
+
+        assertThat(addRegistrationResponse1.getStatus()).isEqualTo(HttpStatus.CREATED_201);
+        final String registrationId1 = UnitTestUtils.getEntityIdFromResponse(addRegistrationResponse1);
+
+        Client client2 = IntegrationTestUtils.buildClient(RULE);
```

Comment:
Definitely not ideal, but this workaround will do. I'll leave a comment for posterity.